bin/faucet/src/server/pow.rs

```diff
@@ -16,10 +19,10 @@ use super::{
     get_tokens::{InvalidRequest, RawMintRequest},
 };
 
-/// The difficulty of the `PoW`.
+/// The maximum difficulty of the `PoW`.
 ///
 /// The difficulty is the number of leading zeros in the hash of the seed and the solution.
-const DIFFICULTY: u64 = 5;
+const MAX_DIFFICULTY: u64 = 6;
```

Comment:
Max difficulty of 6 leading zeros seems pretty low (i.e., it would require, on average, 64 guesses to get the right nonce). A moderately-powerful machine should be able to guess 16 leading zeros in under a second, and 20 leading zeros within a few seconds). I'd probably set the maximum at something like 24.

Also, as mentioned in another comment, ideally, PoW should be based not on the number of leading zeros but on absolute values. This way, it can be adjusted more granularly - but we can do this in a separate PR.